share/completions/systemctl.fish

```diff
@@ -1,8 +1,15 @@
+set -l systemd_version (systemctl --version | head -n 1 | cut -f 2 -d ' ')
```

Comment:
I'd prefer this to be done with `string` - `systemctl --version | string match "systemd*" | string replace -r "\D*(\d+)" '$1'`?